dev/com.ibm.ws.kernel.service/src/com/ibm/ws/common/crypto/CryptoUtils.java

```diff
@@ -392,7 +392,7 @@ public String run() {
         });
     }
 
-    static String getFipsLevel() {
+    public static String getFipsLevel() {
```

Comment:
This likely isn't needed because I don't believe the mock that required this change is necessary.